client/my-sites/importer/section-import.jsx

```diff
@@ -372,8 +368,6 @@ export default connect(
 	( state ) => {
 		const siteID = getSelectedSiteId( state );
 		return {
-			engine: getSelectedImportEngine( state ),
-			fromSite: getImporterSiteUrl( state ),
```

Comment:
Should we add those to the expected `propTypes` now that they're expected to be passed externally?

Reply:
Good idea, added prop types in https://github.com/Automattic/wp-calypso/pull/48413/commits/2adc7da43554c358306768e7b28aabe42b4e2352